api/views/upload_views.py

```
@@ -96,32 +94,33 @@
         try:
             file_path = default_storage.save(unique_filename, uploaded_file)
-            logger.info(f"File saved successfully to: {file_path}")
-        except Exception as e:
-            logger.error(f"Failed to save file: {str(e)}")
+            # logger.info(f"File saved successfully to: {file_path}")
+        except OSError as e:
+
+            # logger.error(f"Failed to save file: {str(e)}")
             return Response(
-                {"error": "Failed to save file to server"},
+                {"error": f"Failed to save file to server {str(e)}"},
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)